ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -664,7 +660,7 @@ def _get_columns(
                                         tagFQN=f"{self.config.tag_category_name}{FQDN_SEPARATOR}{column['policy_tags']}",
                                         labelType="Automated",
                                         state="Suggested",
-                                        source=TagSource.Tag.name,
+                                        source="Tag",
```

Comment:
Let's open an issue to have this directly in the JSON Schema

Reply:
Yes - noted it down

Reply:
will it open it in sometime